.github/workflows/build.yaml

```diff
@@ -26,11 +26,11 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.2.2
+        uses: pypa/cibuildwheel@v2.3.1
         env:
           CIBW_TEST_COMMAND: "pip install -r {project}/requirements/tests.txt && pytest --pyargs uarray"
           # Update according to NEP-29
-          CIBW_BUILD: "{cp3{8..10}-*,pp37-*}"
+          CIBW_BUILD: "{cp3{8..10}-*,pp38-*}"
```

Comment:
I think `python_requires` in `setup.py` needs updated as well. It says 3.6 currently.